package.json

```diff
@@ -18,7 +20,7 @@
     "react-landing-page": "^2.0.2",
     "react-redux": "^7.1.3",
     "redux": "^4.0.5",
-    "rich-markdown-editor": "https://github.com/fydp-saac/rich-markdown-editor.git",
+    "rich-markdown-editor": "github:fydp-saac/rich-markdown-editor#master",
```

Comment:
does this allow the editor plugin to be pulled from git rather than local?

Reply:
nope, for some reason it doesnt work. i also added a post install script and that didnt work too :(